src/newSeed/baseStage.scss

```diff
@@ -60,9 +57,9 @@
   label {
     display: block;
     font-size: 1.1em;
+    font-weight: bold;
     line-height: 2em;
-    color: $White;
-    @include standardTextGradient;
+    color: $Secondary02;
```

Comment:
does this enable is to just include colors.scss instead of styles.scss?

Reply:
I don't understand but the labels colors were not gradients but just block plain colors hence I removed the text gradient mixin

Reply:
can you please try importing colors instead styles at the top of this file, see if it compiles?

Reply:
it does compile